lti_consumer/api.py

```diff
@@ -278,7 +280,33 @@ def validate_lti_1p3_launch_data(launch_data):
                 f"The context_type attribute {context_type} in the launch data is not a valid context_type."
             )
 
+    proctoring_launch_data = launch_data.proctoring_launch_data
+    if (launch_data.message_type in ["LtiStartProctoring", "LtiEndAssessment"] and not
+            launch_data.proctoring_launch_data):
```

Comment:
you clearly meant to also make this just proctoring_launch_data for brevity and missed it